src/integrations/slack/conversations.ts

```
@@ -154,3 +155,42 @@ export const isBotAllowed = (
+    logger.warn("No channel ID provided to getConversationName");
+    return "";
+  }
+
+  try {
+    logger.info("Resolving channel name for ID", channelId);
+    const response: ConversationsInfoResponse =
+      await slackClient.conversations.info({
+        channel: channelId,
+      });
+
+    if (response.channel && response.channel.name) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)